miniserver/app.js

```diff
@@ -0,0 +1,144 @@
+require('dotenv').config()
+const createError = require('http-errors')
+// const rateLimit = require('express-rate-limit')
+const Fingerprint = require('express-fingerprint')
+const express = require('express')
+const path = require('path')
+const cookieParser = require('cookie-parser')
+const logger = require('morgan')
+const config = require('./config')
+const _index = require('./routes/index')
+const bodyParser = require('body-parser')
+const app = express()
+const https = require('https')
+const http = require('http')
+const env = process.env.NODE_ENV || 'development'
+const fs = require('fs')
+const blockchain = require('./blockchain')
+
+const swaggerUI = require('swagger-ui-express')
```

Comment:
Let's host this in a separate server to minimize security exposure, or generate static pages and deploy on file buckets.

Reply:
removed can create separate server under new ticket

Reply:
Tracked ini #13